server/src/main/java/io/unitycatalog/server/persist/dao/ModelVersionInfoDAO.java

```diff
@@ -78,7 +78,7 @@ public static ModelVersionInfoDAO from(ModelVersionInfo modelVersionInfo) {
         .updatedAt(
             modelVersionInfo.getUpdatedAt() != null
                 ? new Date(modelVersionInfo.getUpdatedAt())
-                : new Date())
+                : null)
```

Comment:
Why is this different from the logic for `createdAt`?

Reply:
This was changed to match the other DAO's filling in of this field.  I have confirmed that the field is always being filled in at creation/update, so there's little risk in having this match the other DAOs